src/BoundaryConditions/field_boundary_conditions.jl

```diff
@@ -277,43 +277,27 @@ regularize_south_boundary_condition(bc::DefaultBoundaryCondition, grid::Latitude
     regularize_boundary_condition(default_prognostic_bc(grid, Val(:south), loc, bc), grid, loc, args...)
 
 function default_prognostic_bc(grid::LatitudeLongitudeGrid, ::Val{:north}, (LX, LY, LZ), default)
-    if LY == Nothing
-        return nothing
-    end
-
     φnorth = @allowscalar φnode(grid.Ny+1, grid, Face()) 
     cca_loc = LX == Center && LY == Center # scalar
     default_bc = default_prognostic_bc(topology(grid, 2)(), LY(), default)
-    return φnorth ≈ 90 && cca_loc ? PolarBoundaryCondition(grid, :north, LZ) : default_bc
+    return φnorth ≈ 90 && cca_loc ? PolarBoundaryCondition(grid, :north, LY(), LZ) : default_bc
```

Comment:
why is it `LY(), LZ`, should we have `LY(), LZ()`

Is it necessary to check `cca_loc` after this change?